chrysler_pacifica_ehybrid_s_2019/drivers.launch

```diff
@@ -83,8 +84,26 @@ If not using simulated drivers they are activated if the respective mock argumen
     <arg name="auth_config_file" value= "$(arg vehicle_calibration_dir)/lightbar/auth_config.yaml"/>
   </include>
 
-  <!-- TODO AVT Vimba Camera Left Driver Node -->
-  <!-- TODO AVT Vimba Camera Right Driver Node -->
+  <!-- AVT Vimba Camera Left Driver Node -->
+  <include if="$(arg avt_vimba_camera)" file="$(find avt_vimba_camera)/launch/mono_camera.launch">
+	<arg name="guid" default=""/>
+ 	<arg name="ip" default="192.168.30.10"/>
+    <arg name="camera_info_url" default="file://$(find vehicle_calibration_dir)/avt_vimba_camera/calibration_50-0536911040.yaml"/>
```

Comment:
Again you need to stop using the find syntax and use the vehicle calibration directory argument instead